cypress-shared/support/fedex-shipping/graphql_testcase.js

```diff
@@ -56,19 +70,28 @@ export function savePackingOptimizationAppSetting(settings) {
 }
 
 export function updateDockConnection(id, remove = false) {
+  const queryVariables = { dockId: id, toRemove: remove }
+  cy.qe('Update dock connection via graphQl.')
+
   const query =
     'mutation' +
     '($dockId: String, $toRemove: Boolean)' +
     '{updateDockConnection(updateDock: {dockId:$dockId,toRemove:$toRemove})}'
+  cy.qe(`The graphql query is,${query}`)
+  cy.qe(`Variables - ${queryVariables}`)
```

Comment:
@thyarles suggested to not print variables since it may have sensitive information